3drepo/3drepo.io
# ISSUE #4639 - Sequences don't revert to a previously selected date on cancel

This fixes #4639 

#### Description
This bug was being caused by the mui-lab calendar
Instead of updating muilab or using the x-date-pickers it seemed like a good reason to change to the better looking v5 component


#### Test cases
- Open a model with a sequence
- Click the sequence date input to bring up the datepicker.
- Select a date with it
- Using the arrows navigate the frames
- Open the datepicker again but this time cancel
- The date should not have changed (before the date was reverting to the previously selected one)
